website/blog/2023-06-29-databend-v1.2-release.md

```diff
@@ -0,0 +1,316 @@
+
+In addition, we have designed and implemented enterprise features such as **Computed Columns**, **VACUUM TABLE**, and **Serverless Background Service**. If you are interested in these features, you can contact [Databend Support](https://www.databend.com/contact-us) for upgrade information or visit [Databend Cloud](http://app.databend.cn/) for an instant experience.
+
+## New Features and Enhancements
+
+Discover Databend's new features and find your perfect fit with a quick overview.
+
+### New Data Type: `BITMAP`
+
+Databend has added support for the `BITMAP` datatype and implemented a series of related functions.
+
+`BITMAP` is a type of compressed data structure that can be used to efficiently store and manipulate sets of boolean values. It has wide applications in data analysis and querying, providing fast set operations and aggregation capabilities. Common use cases include Distinct Count, Filtering & Selection, and Compressed Storage.
+
+Our implementation of the BITMAP data type utilizes `RoaringTreemap`. Using this data structure brought us improved performance and decreased memory usage in comparison to alternative bitmap implementations.
+
+```sql
+SELECT user_id, bitmap_count(page_visits) AS total_visits
+FROM user_visits
+
++--------+------------+
+|user_id |total_visits|
++--------+------------+
+|       1|           4|
+|       2|           3|
+|       3|           4|
++--------+------------+
+```
+
+If you are interested in learning more, please check out the resources listed below:
+
+- [Docs |  Data Types - BITMAP](https://databend.rs/doc/sql-reference/data-types/data-type-bitmap)
+- [Docs | SQL Functions - Bitmap Functions](https://databend.rs/doc/sql-functions/bitmap-functions/)
+- [Paper | Consistently faster and smaller compressed bitmaps with Roaring](https://arxiv.org/pdf/1603.06549.pdf)
+
+### Direct Query of CSV/TSV/NDJSON Files Using Column Position
+
+To query files such as CSV/TSV/NDJSON that do not have a schema, they had to be loaded into a table before querying. However, sometimes users do not know the specific details of the file in advance (such as how many columns a CSV file has), or they just want to make temporary queries.
+
+To address this, Databend introduces **Column Position** and uses the `$N` syntax to represent the Nth column. All columns in CSV/TSV files are treated as `String` type. If the number of columns in a row is less than the column number used, it will be padded with an empty string. NDJSON files have only one column `$1`, which is of type `Variant` .
```

Comment:
```suggestion
To make it easier, Databend introduces **Column Position** and uses the `$N` syntax to represent the Nth column. All columns in CSV/TSV files are treated as `String` type. If the number of columns in a row is less than the column number used, it will be padded with an empty string. NDJSON files have only one column `$1`, which is of type `Variant` .
```